pm_nq/test-data/tree.txt

```diff
@@ -1,3 +1,4 @@
+conda_activate.log
```

Comment:
is this addition correct?

Reply:
Apparently not - when I run the tests locally it was generating `conda_activate.log` (I don't know why) so I needed to add it to get the tests to pass. But the GHA tests don't generate it `conda_activate.log` so I've removed it.